src/Site.js

```diff
@@ -94,9 +95,19 @@ const FOOTER_DEFAULT = '<footer>\n'
   + '  </div>\n'
   + '</footer>\n';
 
+const HEADER_DEFAULT = '<header>\n'
+  + '  <div class="bg-primary display-4 text-center text-white">\n'
```

Comment:
This is subjective, but the default header generated seems to be too dominating compared to the rest of the parts of the webpage. :P

This is how it looks like now with a `markbind init` website:

<img src="https://user-images.githubusercontent.com/3168908/54742184-9f1f1680-4bfb-11e9-9e83-8c498073471a.png" width=500/>

Reply:
Intended to be a large banner / Jumbotron. Similar to the one in [Markdown Guide](https://www.markdownguide.org/cheat-sheet/)